packages/rn-tester/android/app/src/main/gen/com/facebook/react/uiapp/BuildConfig.java

```diff
@@ -0,0 +1,8 @@
+/*___Generated_by_IDEA___*/
```

Comment:
What are these files?

Reply:
These weren't supposed to be committed. Removing them!